web-ng/src/app/components/feature-panel/feature-panel.component.html

```diff
@@ -28,11 +28,21 @@
       </mat-card-subtitle>
     </mat-card-header>
     <mat-card-content>
-      <div>
         <div *ngFor="let field_entry of observation.form.fields ">
-          <label class="field-label">{{ field_entry[1].label.get(lang) }}</label>
-          <div class="field-response">{{ observation.responses?.get(field_entry[1].id)?.value }}</div>
-        </div>
+          <div *ngIf="field_entry[1].type == fieldTypes.TEXT">
+            <label class="field-label">{{ field_entry[1].label.get(lang) }}</label>
```

Comment:
is there any reason we are target index 1 instead of i?

Reply:
this is because immutable Map when iterating in ngFor is returning entry that is an array with 2 elements. In the future we will use precalculated list for that because we will need to sort entries anyway.

I am also considering changing this Map into a List in a model itself because I do not think that we ever need to access a field by id. We usually just iterate fields in the form and access Response by id.